config/nrfconnect/chip-module/Kconfig.defaults

```diff
@@ -167,6 +167,17 @@ config BT_DEVICE_NAME_GATT_WRITABLE
 config CHIP_NFC_COMMISSIONING
     default y
 
+# Enable Factory Data support by default
+
+config CHIP_FACTORY_DATA
```

Comment:
It may cause trouble to customers who include `Kconfig.defaults` but do not have the factory data partition. Not sure if you considered that and accepted the risk, but just saying :)